lib/overrides.js

```diff
@@ -13,11 +13,9 @@ function* walkAddonTree(project, pathToAddon = []) {
   }
 }
 
-function requirementFor(pkg, deps = {}) {
-  return deps[pkg];
-}
+const requirementFor = (pkg, deps = {}) => deps[pkg];
```

Comment:
The same goes for this—the arrow function is cleaner-looking, but it does actually have different semantics. (It can't be referred to *earlier* in the file, now, since it isn't hoisted!)

Reply:
Thanks dude for be so cool about these changes, and being helpful by helping people to understand what's more important and what's not, we need more people like you in open source,